source/js/main.js

```diff
@@ -59,6 +60,10 @@ let main = {
       ReactDOM.render(<People/>, document.getElementById(`people`));
     }
 
+    if (document.getElementById(`people-box`)) {
```

Comment:
is `people-box` a good descriptive name? should we rename it to `featured-people-box` or something like that?